executor/index_merge_reader_test.go

```diff
@@ -797,3 +797,19 @@ func TestIntersectionMemQuota(t *testing.T) {
 	err := tk.QueryToErr("select /*+ use_index_merge(t1, primary, idx1, idx2) */ c1 from t1 where c1 < 1024 and c2 < 1024")
 	require.Contains(t, err.Error(), "Out Of Memory Quota!")
 }
+
+func TestIndexMergeResultChPanic(t *testing.T) {
```

Comment:
maybe add more failpoint tests that simulate panic happens in the goroutine of all the kinds of workers?

Reply:
Done, Add 6 failpoints.